bears/apertium/ApertiumLintBear.py

```diff
@@ -0,0 +1,363 @@
+import os
+import json
+import lxml
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='apertium_lint',
+        output_format='regex',
+        output_regex=r'(?P<severity>\w+) : (?P<message>[^:\n]+: '
+                     r'(?P<origin>[^\n]+))\n[^:\n]+: (?P<line>\d+)',
+        config_suffix='.json',
+        severity_map={'Warning': RESULT_SEVERITY.MAJOR})
+class ApertiumLintBear:
+    """
+    `Apertium lint` is a python module that lints out irregular yet
+    acceptable constructs that may creep into files involved in the
+    transfer process. The lint is designed specifically to handle
+    4 file types dictionaries, transfer, modes and tagger.
+
+    Lints the file using
+    `apertium_lint <https://svn.code.sf.net/p/apertium/svn
+    /trunk/apertium-tools/apertium-lint/>`.
+    """
+    LANGUAGES = {'Apertium'}
+    REQUIREMENTS = {PipRequirement('apertium-lint', '0.29'),
+                    PipRequirement('lxml', '3.6.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Redundancy', 'Syntax', 'Formatting', 'Unused Items',
```

Comment:
I think it would be better to call "Unused Items" -> "Unused Code" so it is consistent with everything else.
Maybe this aspect should be renamed "Unused Element" to be more generic.
Could you discuss this with the aspects team . cc @userzimmermann